packages/utils/src/useExport/useExportWithItems.js

```diff
@@ -0,0 +1,20 @@
+import { exportableColumns } from './helpers';
+
+import useExport from './useExport';
+
+export const useExportWithItems = (items, columns, options = {}) => {
```

Comment:
Since this function doesn't use any of the React's lifecycle let's not call it `useExportWithItems` it's just a helper function after all.